File: 3875-construct-uniform-parity-array-i/3875-construct-uniform-parity-array-i.cpp
class Solution {
public:
    bool uniformArray(vector<int>& nums1) {
        int oddcount = 0, evencount = 0;
        for (int x : nums1) {
            if (x % 2 == 0) evencount++;
            else oddcount++;
        }
        if (oddcount == nums1.size() || evencount == nums1.size())
            return true;
        return true;
    }
};

File: 3875-construct-uniform-parity-array-i/3875-construct-uniform-parity-array-i.java
class Solution {
    public boolean uniformArray(int[] nums1) {
        int oddcount=0;
        int evencount=0;
        for(int num:nums1){
            if(num%2==0){
                evencount++;
            }
            oddcount++;
        }
        if(oddcount==nums1.length||evencount==nums1.length){
            return true;
        }
        return true;
    }
}